### DISCUSSION TITLE
crop + padding

### DISCUSSION THREAD

**Discussion by CA10C3SAR** [ImageMagick/ImageMagick#2232] [Help] [answered]
how I could crop each slot of this image correctly?
![backpack_area](https://user-images.githubusercontent.com/6724006/86506105-ce32aa80-bda2-11ea-8f10-b3935975dd31.png)

what i tried:
convert y.png -crop 4x5-0-0@ +repage +adjoin y2.png

result:
![12](https://user-images.githubusercontent.com/6724006/86506132-fb7f5880-bda2-11ea-844d-ff2317d44a29.png)
![4](https://user-images.githubusercontent.com/6724006/86506133-fc17ef00-bda2-11ea-9a37-f07fb1ef1abb.png)

the result is not cropped correctly, the image has 3x3 of padding
![image](https://user-images.githubusercontent.com/6724006/86506152-236ebc00-bda3-11ea-8f94-b9d2d49025f4.png)

correct would be 20 pieces of 34x34 like this:
![image](https://user-images.githubusercontent.com/6724006/86506191-89f3da00-bda3-11ea-98f8-6fd87c664686.png)

**Comment by snibgo** [comment, marked as answer]
You can crop it into 37x37 pieces. Then crop each into 34x34. Naming your input "4x5tiles.png":
```
magick 4x5tiles.png -crop 37x37 +repage -crop 34x34+0+0 +repage 4x5t-%02d.png
```
